### DISCUSSION TITLE
After adding some accounts now I want to add verification flow

### DISCUSSION THREAD

**Discussion by shreeharsha-factly** [ory/kratos#2085] [Q&A] [answered]
My earlier accounts didn't have a verification flow. Now I want to add verification flow. How to initiate verification flow for existing users?

**Comment by aeneasr** [comment, marked as answer]
You should be able to point them to the verify endpoint, and then ask them to verify. Or, enable "require_verified_address" for sign in using the hooks! That would also ask them to verify.

**Reply by vinckr** [reply]
Let me know if you had success with these methods @shreeharsha-factly , I think this could be added to the documentation.

**Reply by vsumit89** [reply]
Hey @vinckr I work with @shreeharsha-factly, so what we did is that we added the required_verified_address hook into our application, if they are not verified they will be basically shown a button which will redirect them to the verification flow in our application which is similar to  [this](https://kratos-reference-ui-react-nextjs.vercel.app/verification) where they can verify their email address. After verifying their email address they can sign in.

**Reply by vinckr** [reply]
Thanks for coming back to us with the solution 🙏 I will see to make good use of it.
